ui/packages/catalog-ui-search/src/main/webapp/component/search-form/forms-sharing/search-form-sharing.collection.view.js

```diff
@@ -12,27 +12,27 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
- /*global require*/
- const Marionette = require('marionette');
- const _ = require('underscore');
- const $ = require('jquery');
- const SearchFormView = require('../search-form.view');
- const SearchFormSharingCollection = require('./search-form-sharing.collection');
- const CustomElements = require('js/CustomElements');
+/* global require */
+const Marionette = require('marionette')
+const _ = require('underscore')
+const $ = require('jquery')
+const SearchFormView = require('../search-form.view')
+const SearchFormSharingCollection = require('./search-form-sharing.collection')
+const CustomElements = require('js/CustomElements')
 
- module.exports = Marionette.CollectionView.extend({
-     childView: SearchFormView,
-     className: 'is-list is-inline has-list-highlighting',
-     initialize: function(options) {
-        let searchFormSharingCollection = new SearchFormSharingCollection();
-        this.collection = searchFormSharingCollection.getCollection();
-        this.searchFormSharingCollection = searchFormSharingCollection;
-        this.options = options;
-     },
-     childViewOptions: function() {
-        return {
-            queryModel: this.options.model,
-            sharingLightboxTitle: 'Search Form Sharing'
-        };
-     },
- });
\ No newline at end of file
+module.exports = Marionette.CollectionView.extend({
+  childView: SearchFormView,
+  className: 'is-list is-inline has-list-highlighting',
+  initialize: function (options) {
+    let searchFormSharingCollection = new SearchFormSharingCollection()
```

Comment:
✏️ use `const` if in doubt and let webpack yell at you if it's wrong 😄

Reply:
✏️ consider replacing this with `this.searchFormSharingCollection = new SearchFormSharingCollection()`